x-pack/elastic-agent/_meta/config/common.reference.p2.yml.tmpl

```diff
@@ -131,6 +131,13 @@ inputs:
 #   # period define how frequent we should look for changes in the configuration.
 #   period: 10s
 
+# Load input configuration from external configuration files when running in standalone mode.
+# Each file matching the glob must contain a list of one or more input definitions. 
+# The first line of each external configuration file must be an input definition
+# that starts with `- type`.
+# agent.config.inputs:
```

Comment:
Do we need to make the path configurable? Can't we just rely on a predefined directory an pattern?

Reply:
It is possible. I chose to make it configurable because it is configurable in Filebeat.

Reply:
If possible, let's simplify. Sometimes less is more.

Reply:
So if I understood correctly, you do not want a configuration option at all, but rather load files automatically if they are placed under a predefined directory e.g. `{path.config}/inputs.d`, right?

Reply:
I hardcoded the glob of the external configurations to `{path.config}/inputs.d/*.yml`.

Reply:
Exactly. Lets remove as many config options as we can at first and only add them if we have a strong use case for these.